### PR TITLE
 Adding new classes for test

### PR DESCRIPTION
**Pull request by h-haddad**

### CODE REVIEW

File: galite-tests/src/test/kotlin/org/kopi/galite/tests/examples/Common.kt [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  val refTraining = integer("TRAINING_REFERENCE").references(Training.id)
+
+  override val primaryKey = PrimaryKey(id, name = "PK_CENTER_ID")
+}
+
+object Type : CodeDomain<String>() {
+  init {
+    "JAVA" keyOf "JAVA"
+    "PYTHON" keyOf "PYTHON"
+    "HTML" keyOf "HTML"
```
Comment (hfazai) [inline galite-tests/src/test/kotlin/org/kopi/galite/tests/examples/Common.kt:154]: Create an example where key is not the same as its value

File: galite-tests/src/test/kotlin/org/kopi/galite/tests/examples/Common.kt [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  val price = decimal("UNIT_PRICE", 9, 3)
+  val active = bool("ACTIVE",)
+  val photo = blob("PHOTO").nullable()
+  val informations = varchar("INFORMATION", 200).nullable()
+
+  override val primaryKey = PrimaryKey(id, name = "PK_TRAINING_ID")
+}
+
+object Center : Table("Center") {
+  val id = integer("ID")
```
Comment (hfazai) [inline galite-tests/src/test/kotlin/org/kopi/galite/tests/examples/Common.kt:138]: You can use `IntIDTable` instead

File: galite-tests/src/test/kotlin/org/kopi/galite/tests/examples/TrainingR.kt [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    align = FieldAlignment.LEFT
+  }
+  val disponibility = field(Domain<Boolean>(2)) {
+    label = "disponibility"
+    help = "disponibility"
+    align = FieldAlignment.LEFT
+  }
+
+
+
```
Comment (hfazai) [inline galite-tests/src/test/kotlin/org/kopi/galite/tests/examples/TrainingR.kt:182]: remove empty lines

File: galite-tests/src/test/kotlin/org/kopi/galite/tests/examples/Common.kt (hunk 2 above)
Comment (h-haddad) [reply galite-tests/src/test/kotlin/org/kopi/galite/tests/examples/Common.kt:138]: i tried it but i have a problem when i need to use foreign key

**Review by hfazai — APPROVED** [review]

**Pull request merged**